cfme/tests/services/test_rest_services.py

```diff
@@ -778,3 +778,77 @@ def test_edit_orchestration_templates(self, rest_api, orchestration_templates, f
             assert edited[i].description == new[i]['description']
             orchestration_templates[i].reload()
             assert orchestration_templates[i].description == new[i]['description']
+
+    @pytest.mark.tier(3)
+    @pytest.mark.uncollectif(lambda: version.current_version() < '5.8')
+    @pytest.mark.parametrize(
+        "from_detail", [True, False],
+        ids=["from_detail", "from_collection"])
+    def test_copy_orchestration_templates(self, request, rest_api, orchestration_templates,
+            from_detail):
+        """Tests copying of orchestration templates.
+
+        Metadata:
+            test_flag: rest
+        """
+        response_len = len(orchestration_templates)
```

Comment:
**Optional**
I don't quite know why but this name bothers me. I think because its not a response that you've generated here, but in the fixture. consider `num_orch_templates` or something like that. Or not, its your variable :)